recipes/_master_package.rb

```diff
@@ -46,13 +46,32 @@
 when 'rhel'
   include_recipe 'yum::default'
 
+  ruby_block "Security off" do
+    block do
+      Chef::Log.info "Waiting until Jenkins config.xml file is present"
+      until File.exist?("#{node['jenkins']['master']['home']}/config.xml")
+        sleep 1
+        Chef::Log.debug('.')
+      end
+
+      fe = Chef::Util::FileEdit.new("#{node['jenkins']['master']['home']}/config.xml")
+      fe.search_file_replace_line(/  <useSecurity>true<\/useSecurity>/,
+                                  "  <useSecurity>false<\/useSecurity>")
+      fe.write_file
+    end
+    action :nothing
+    notifies :restart, 'service[jenkins]', :immediately
+  end
+
   yum_repository 'jenkins-ci' do
     baseurl node['jenkins']['master']['repository']
     gpgkey  node['jenkins']['master']['repository_key']
   end
 
   package 'jenkins' do
     version node['jenkins']['master']['version']
+    notifies :restart, 'service[jenkins]', :immediately
```

Comment:
is this actually needed as `ruby_block[Security off]` is restarting `service[jenkins]` anyway?